Add currentCapacity to ui for workerpools

imbstack opened the pull request:
Github Bug/Issue: Fixes #2882 

As it says on the tin! Doesn't fix all of the issues in the issue but covers the most helpful ones IMHO

helfi92 commented on services/web-server/src/graphql/WorkerManager.graphql:
```diff
@@ -5,6 +5,7 @@ type WorkerManagerWorkerPoolSummary {
     providerId: String!
     config: JSON
     emailOnError: Boolean!
+    currentCapacity: Int!
```
Navigating to http://localhost:5080/worker-manager shows a red banner with text `Cannot return null for non-nullable field WorkerManagerWorkerPoolSummary.currentCapacity.`

Is this value not always defined?

imbstack replied:
This will only work once the services with `currentCapacity` are deployed. It works when I run it against my dev deployment

helfi92 replied:
Makes sense. Thank you!

imbstack commented:
Sweet, thanks for the pointers. I've updated it to look like that and added a couple links like the issue wanted. Feel free to have me solve this in a different way though!

helfi92 commented on ui/src/components/WMWorkerPoolEditor/index.jsx:
```diff
@@ -315,10 +332,57 @@ export default class WMWorkerPoolEditor extends Component {
+                className={classes.metadata}
+                icon={<WorkerIcon />}
+                label="Workers (Queue View)"
+                component="a"
```
We have a custom component "Link" that allows prefetching the view before the user clicks on it. It will help make the transition much faster. We would need to also change the `href` to `to`.

helfi92 replied:
See the diff :)

imbstack commented:
Oh interesting. The capacity work has been merged for a while. Does it need to be deployed to production too before we land this?

helfi92 commented:
> Oh interesting. The capacity work has been merged for a while. Does it need to be deployed to production too before we land this?

Ah, yes, sorry. Ship it :)
